### ISSUE TITLE
Can FairMOT directly detect vehicles without training

### ISSUE THREAD

**Issue by Ckq-Sugar** [CaptainEven/FairMOTVehicle#4]
Hi !
I am a novice in MOT，I have a questuon?
Can I directly use the original model(FairMOT) to detect the vehicle?

**Comment by CaptainEven** [comment]
@Ckq-Sugar Of course you can, some scripts to prepare for training data added in this repo compared to FairMOT such as script to convert UA_DETRAC dataset.

**Comment by Ckq-Sugar** [comment]
@CaptainEven  can you tell me details about modifying the FairMOT ,so that FairMOT can test the vehicle?
thank you very much!

**Comment by CaptainEven** [comment]
You need to train a new MOT tracker to track vehicles, for details, you can just refer to the readme of this repo. Am i miss-understand you by "original model"(Is it refered to a trained checkpoint file .pth or just use original FairMOT to train a vehicle mot tracker)?

**Comment by Ckq-Sugar** [comment]
@CaptainEven The original model which I mentioned above is FairMOT 's Pretrained models Baseline model.  
so if I  only want detect and track vehicles without thinking accuracy, I can use  FairMOT 's Pretrained models Baseline model?

**Comment by CaptainEven** [comment]
@Ckq-Sugar I'm afraid not. Because the pre-trained model of FairMOT is just for person MOT tracking.

**Comment by Ckq-Sugar** [comment]
OK,I do know. thank you!
 I also am interested in you reps (MCMOT and FairMOTVehicle)
I am tring to reproduce your MCMOT, I find that I dont download model when run model_down_script.sh.
could you provide other links?
if you provide it ,I will be very grateful！！！

**Comment by CaptainEven** [comment]
The details of MCMOT training including dataset preparation will be released in the future.
